src/main/java/Entities/Player.java

```diff
@@ -26,26 +30,58 @@ public Player(String name) {
         this.position = 0;
     }
 
-
+    /**
+     * This basically sets the player attribute of inJail, which in this case is a boolean value
+     * @param inJail the parameter for this is a boolean value that sets their inJail property to true or false
+     */
     public void setInJail(boolean inJail) {
         this.inJail = inJail;
     }
+
+    /**
+     * This sets the position of the player
+     * @param position the position of the player should be between 0 and 39 inclusive, as this dictates where on the
+     *                 board the player is
+     */
     public void setPosition(int position) {
         this.position = position;
     }
 
-
+    /**
+     * This sets the number of get out of jail free cards the player has that was drawn from the community chest or
+     * chance cards
+     * @param num this parameter is the number of cards to increment by for player
+     */
     public void setJailCards(int num) {
```

Comment:
I talked to Derrick and Susannah and we will be removing this method along with some other redundant ones as they won't be needing them to implement their classes.